specification/impact/Impact.Management/connectors.tsp

```diff
@@ -22,6 +22,23 @@ model Connector is ProxyResource<ConnectorProperties> {
   name: string;
 }
 
+@doc("A connector is a resource that can be used to proactively report impacts against workloads in Azure to Microsoft.")
+model ConnectorUpdate is ProxyResource<ConnectorPropertiesUpdate> {
+  @doc("The name of the connector")
+  @pattern("^[a-zA-Z0-9-]{3,24}$")
+  @key("connectorName")
+  @segment("connectors")
+  @path
+  name: string;
+}
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "No ProvisioningState"
+@doc("Details of the Connector.")
+model ConnectorPropertiesUpdate {
+  @doc("connector type")
+  connectorType: Platform;
+}
```

Comment:
Should you use some template (maybe `ResourceUpdateModel`), avoid service define the same thing twice?

If you ask service to dupicate things, you need have them in review.

Reply:
I updated to `ResourceUpdateModel` to reduce the duplications. And it would introduce two changes with original swagger, 1) we lose the `allOf` information because in new update model we would spread all properties into one model. and 2) the naming convention is changed from `ConnectorPropertiesUpdate` to `ConnectorUpdateProperties`.